scripts/download_data.py

```diff
@@ -280,14 +293,36 @@ def downloadFastas(self):
                 'display': 'fasta',
                 'range': '{}-{}'.format(0, maxPos)})
             url = '{}%26{}'.format(path, args)
+            tempFileName = '{}.fa.temp'.format(chromosome)
             fileName = '{}.fa'.format(chromosome)
-            downloader = utils.HttpFileDownloader(url, fileName)
+            downloader = utils.HttpFileDownloader(url, tempFileName)
             downloader.download()
+            # We need to replace the header on the downloaded FASTA
+            with open(tempFileName, "r") as inFasta,\
```

Comment:
Haha, I knew you were going to bring the puppies into this! It turns out that this is the one example in PEP8 where using backslashes is recommended. You can't use brackets in this context for some reason.